doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1570,6 +1576,11 @@ An :token:`s_item` can simplify the set of subgoals or the subgoals themselves:
   tactic ``simpl`` [#5]_.
 + ``//=`` combines both kinds of simplification; it is equivalent to
   ``/= //``, i.e., ``simpl; try done``.
++ the user can define variants of the above and access them by number.
+  In particular ``/1/`` executes ``try ssrdone1``, ``/1=`` executes
+  ``ssrsimpl1`` and ``/2/1=`` executes ``ssrsimpl1; try ssrdone2``.
+  ``ssrdone0`` is defined in |SSR| as a variant of :tacn:`done` not
+  calling :tacn:`split`.
```

Comment:
Give a concrete example where such tactics are defined. It will also help see the use case.